packages/trader/src/Modules/Contract/Components/LastDigitPrediction/digit-display.tsx

```diff
@@ -1,14 +1,39 @@
 import classNames from 'classnames';
 import { observer } from 'mobx-react';
-import PropTypes from 'prop-types';
 import React from 'react';
 import { DesktopWrapper } from '@deriv/components';
 import { isMobile } from '@deriv/shared';
 import { Bounce } from 'App/Components/Animations';
-import Digit from './digit.jsx';
-import DigitSpot from './digit-spot.jsx';
-import LastDigitStat from './last-digit-stat.jsx';
+import Digit from './digit';
+import DigitSpot from './digit-spot';
+import LastDigitStat from './last-digit-stat';
+import { ProposalOpenContract } from '@deriv/api-types';
 
+type TDigitDisplay = {
```

Comment:
@henry-deriv let's extend this type instead of adding cross-dependency on other types inside of it. Otherwise, it will be very hard to track and maintain.
```suggestion
type TDigitDisplay = TDigit & TLastDigitStat & {
    //... prop types excluding the ones from TDigit and TLastDigitStat
```
please use this approach in the rest of the files🙂